src/shims/unix/linux/epoll.rs

```diff
@@ -168,25 +266,56 @@ pub trait EvalContextExt<'tcx>: crate::MiriInterpCxExt<'tcx> {
     fn epoll_wait(
         &mut self,
         epfd: &OpTy<'tcx>,
-        events: &OpTy<'tcx>,
+        events_op: &OpTy<'tcx>,
         maxevents: &OpTy<'tcx>,
         timeout: &OpTy<'tcx>,
     ) -> InterpResult<'tcx, Scalar> {
         let this = self.eval_context_mut();
 
         let epfd = this.read_scalar(epfd)?.to_i32()?;
-        let _events = this.read_scalar(events)?.to_pointer(this)?;
-        let _maxevents = this.read_scalar(maxevents)?.to_i32()?;
-        let _timeout = this.read_scalar(timeout)?.to_i32()?;
+        let maxevents = this.read_scalar(maxevents)?.to_i32()?;
```

Comment:
Add a test to see what happens when someone sends a negative number of `maxevents`

Reply:
it should hit the `unwrap` below, so we need to turn it into an unsup or depending on what the docs say, a normal libc error

Reply:
Ah the libc documentation said if maxevents <= 0, we need to throw ``EINVAL``